modules/sdk-coin-ton/src/lib/transactionBuilderFactory.ts

```diff
@@ -19,6 +20,13 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
     return new TransferBuilder(this._coinConfig);
   }
 
+  /**
+   * Returns a specific builder to create a TON withdraw transaction
+   */
+  getWithdrawBuilder(): WithdrawBuilder {
```

Comment:
can you change the name to SingleNominatorWithdrawBuilder. Withdraw is too broad